sdk/cosmos/azure-cosmos-kafka-connect/CHANGELOG.md

```diff
@@ -1,14 +1,6 @@
 ## Release History
 
-### 2.4.0-beta.1 (Unreleased)
-
-#### Features Added
-
-#### Breaking Changes
-
-#### Bugs Fixed
-
-#### Other Changes
+### 2.4.0 (2025-05-13)
```

Comment:
Changelog needs some entries - CosmosPagedIterable for example

Reply:
Fixed